### PR TITLE
fix(lib): format characters

### PR DESCRIPTION
**Pull request by safl**

### CODE REVIEW

File: lib/xnvme_be_windows_async_ioring.c [hunk 1]
```diff
@@ -36,7 +36,7 @@ xnvme_be_windows_ioring_init(struct xnvme_queue *q, int XNVME_UNUSED(opts))
 
 	if (!handle) {
 		err = GetLastError();
-		XNVME_DEBUG("Failed creating IO ring handle: 0x%x\n", err);
+		XNVME_DEBUG("Failed creating IO ring handle: %" PRId32, err);
```
Comment (NaddiNadja) [inline lib/xnvme_be_windows_async_ioring.c:39]: There was a line-break `\n` at the end of this string before (and in many of the other places `PRId32` has been inserted) - is it OK that it's gone now?

**Review by NaddiNadja — COMMENTED** [review]
just a single thing I'm unsure of :)

File: lib/xnvme_be_windows_async_ioring.c (hunk 1 above)
Comment (safl) [reply lib/xnvme_be_windows_async_ioring.c:39]: Yeah, they should not be there to begin with since ``XNVME_DEBUG`` already outputs a newline.

**Review by NaddiNadja — APPROVED** [review]
lgtm :)

**Review by karlowich — APPROVED** [review]
lgtm

**Pull request merged**